samples/net/openthread/shell/prj-ot-host.conf

```diff
@@ -55,3 +51,8 @@ CONFIG_OPENTHREAD_SRP_CLIENT=y
 CONFIG_OPENTHREAD_SRP_SERVER=y
 CONFIG_OPENTHREAD_MAC_FILTER=y
 CONFIG_OPENTHREAD_IP6_FRAGM=y
+
+CONFIG_OPENTHREAD_PLATFORM_CSL_UNCERT=20
+CONFIG_IEEE802154_MCXW_CSL_ACCURACY=50
```

Comment:
Driver-specific configs should not be placed in a generic config file,  please put it in board specific config in `boards` subdirectory.